R/download_ts.R

```diff
@@ -38,13 +38,13 @@ download_ts=function(site, variable, destination = NULL, session = NULL, ...){
   }
   
   if(is.null(destination)){
-    destination  = file.path(tempdir(), paste0(paste(site, ts_variable, sep = "_" ), '.', get_ts_extension()))
+    destination  = file.path(tempdir(), paste0(paste(site, ts_variable, sep = "_" ), '.', pkg.env$ts_extension))
   }
 
   #set the intermediate (staging) destination for downloaded gzip file
   if (isGzipped(file_list$url)){
```

Comment:
changed up this stuff in the next PR, since download_ts doesn't actually unzip anymore (that is handled in read_ts, just like zip is in write_ts)